app/src/main/java/com/example/mytrainingpal/components/Break.kt

```diff
@@ -0,0 +1,177 @@
+package com.example.mytrainingpal.screens
+
+import android.annotation.SuppressLint
+import androidx.compose.foundation.Canvas
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.*
+import androidx.compose.runtime.*
+
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.geometry.Size
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.drawscope.Stroke
+import androidx.compose.ui.layout.onSizeChanged
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.IntSize
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.navigation.NavController
+import com.example.mytrainingpal.components.GifImage
+import com.example.mytrainingpal.model.entities.Exercise
+import com.example.mytrainingpal.util.ExerciseDetails
+
+import kotlinx.coroutines.delay
+
+// Timer composable adapted from Phillip Lackners' open source code available on Github:
+// https://github.com/philipplackner/ComposeTimer/blob/master/app/src/main/java/com/plcoding/composetimer/MainActivity.kt
+
+
+@SuppressLint("UnusedMaterialScaffoldPaddingParameter")
+@Composable
+fun Break(
+    navController: NavController,
+    exerciseList: MutableList<Pair<Exercise, ExerciseDetails>>,
+    currentExerciseIndex: Int
+) {
+    var currentExercise: Int = 4;
+    var totalExercises: Int = 10;
+    var totalBreakTimeInSeconds: Int = 5
+
```

Comment:
Your PR looks like it is almost finished 🎉. At first glance, the onliest thing I could spot is that you use constant values for `currentExercise` and `totalExercises`. 
- `currentExercise` should be `currentExerciseIndex`
- `totalExercises` should be `exerciseList.size`
- Having a constant break time is fine, as the settings screen is currently still missing

Reply:
Nice work @raissacavalcanti